code/modules/vehicles/cm_armored.dm

```diff
@@ -357,9 +357,18 @@ var/list/TANK_HARDPOINT_OFFSETS = list(
 	return
 
 /mob/living/tank_collision(obj/vehicle/multitile/hitbox/cm_armored/C, facing, turf/T, turf/temp)
+	if(isdead(src)) //We don't care about the dead
```

Comment:
This check is for dead mobs, such as ghosts. Since this proc is defined for the living, it is not necessary.
I assume you want something like `if(stat == DEAD)`

Reply:
This is done